memberships/views.py

```diff
@@ -181,23 +181,7 @@ def stripe_webhook(request):
             json.loads(request.body), settings.STRIPE_SECRET_KEY
         )
         stripe_webhook = StripeWebhook()
```

Comment:
Do you plan to remove the `StripeWebhook` class and put its functionality into `payments.py` in a separate pull request?

Reply:
Yes, I am working with a fair amount of changes so don't want more stuff being moved around.